.pre-commit-config.yaml

```diff
@@ -28,7 +28,7 @@ repos:
         types: [text]
       - id: sensitive-data-leak-ue-numbers
         name: Sensitive data leak - UE numbers
-        entry: \s([uUeE]{1,2})\d{5,6}\s
+        entry: \b([uUeE]{1,2})\d{5,6}\b
```

Comment:
I think this one is obsolete after https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.generic/pull/189#discussion_r1804589205